--- Makefile	2023-03-20 14:18:15.000000000 +0000
+++ Makefile.new	2023-03-20 14:38:00.000000000 +0000
@@ -66,7 +66,7 @@
 
 JETCDIR = /etc/jove
 DETCDIR = $(DESTDIR)$(JETCDIR)
-JTMPDIR = /var/tmp
+JTMPDIR = $(HOMEDRIVE)/Users/$(USERNAME)/AppData/Local/Temp
 JRECDIR = /var/lib/jove/preserve
 DRECDIR = $(DESTDIR)$(JRECDIR)
 # Install permission for DRECDIR
@@ -380,8 +380,8 @@
 # Thus, if "all" is done first, "install" can be invoked with
 # JOVEHOME pointing at a playpen where files are to be marshalled.
 # This property is fragile.
-install: $(DRECDIR) $(DETCDIR) $(DDOCDIR) $(CMDSDOC) $(TEACHDOC) $(JOVERC) \
-	$(PORTSRVINST) $(RECOVER) $(JOVE) $(TEACHJOVE) $(REFDOC) $(MANUALS)
+install: $(DRECDIR) $(DETCDIR) $(JOVERC) \
+	$(PORTSRVINST) $(RECOVER) $(JOVE) $(TEACHJOVE)
 	@echo See the README about changes to /etc/rc or /etc/rc.local
 	@echo so that the system recovers jove files on reboot after a crash
 
