--- Makefile	2023-03-20 14:18:15.000000000 +0000
+++ Makefile.new	2023-03-20 14:20:32.000000000 +0000
@@ -66,7 +66,7 @@
 
 JETCDIR = /etc/jove
 DETCDIR = $(DESTDIR)$(JETCDIR)
-JTMPDIR = /var/tmp
+JTMPDIR = $(HOMEDRIVE)/Users/$(USERNAME)/AppData/Local/Temp
 JRECDIR = /var/lib/jove/preserve
 DRECDIR = $(DESTDIR)$(JRECDIR)
 # Install permission for DRECDIR
@@ -216,7 +216,7 @@
 TROFFPOST = |gro$(TDEV)
 
 # installed man pages
-MANUALS = $(JOVEM) $(TEACHJOVEM)
+MANUALS =
 
 C_SRC = commands.c commands.tab abbrev.c argcount.c ask.c buf.c c.c case.c jctype.c \
 	delete.c disp.c extend.c fp.c fmt.c insert.c io.c iproc.c \
